ansible/cluster.yml

```diff
@@ -25,10 +25,51 @@
     - nodes
   sudo: yes
   roles:
-    - flannel
+    - { role: flannel, when: networking == 'flannel' }
   tags:
     - flannel
 
+# install opencontrail
+- hosts:
+    - localhost
```

Comment:
This tasks ends up putting a dependency on the running machine to be Linux and in particular certain versions that our `common` and `docker` roles support. If this kind of change is okay with @eparis, then the README  would need some serious changes for this kind of change. 

IMHO this kind of role is something I have tried to stay away from as much as possible.